### ISSUE TITLE
Floating script editor button is hidden when Single Window Mode is enabled

### ISSUE THREAD

**Issue by Goldenlion5648** [godotengine/godot#84435]
### Godot version

4.2 beta4

### System information

Godot v4.2.beta4 - Windows 10.0.19045 - GLES3 (Compatibility) - NVIDIA GeForce GTX 1050 Ti (NVIDIA; 31.0.15.2647) - Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz (8 Threads)

### Issue description

The button to make the script editor floating is no longer present in 4.2 beta 4

4.2:
![image](https://github.com/godotengine/godot/assets/32916571/58f8eaca-14d7-4585-8fcd-c2fdb155059b)

4.1:
![image](https://github.com/godotengine/godot/assets/32916571/6b3df109-28d4-4f16-838a-f5cf642dfb28)

Also using the ctrl shift P shorting and selecting "make floating" under the script editor section does nothing

### Steps to reproduce

Open a script, notice there is no floating button. 

Try to use "make floating" option using ctrl shift P, notice it does nothing

### Minimal reproduction project

N/A

**Comment by Calinou** [comment]
I can see it here in 4.2.beta da0b1eb12 (Linux):

![image](https://github.com/godotengine/godot/assets/180032/b00b9031-5ec9-498b-b45b-7135064907ff)

Make sure **Single Window Mode** is unchecked in the Editor Settings, or this option will be unavailable. Perhaps we should disable the icon with a tooltip instead of hiding it entirely.

**Comment by Goldenlion5648** [comment]
You are correct, disabling single window mode reveals the floating button, and it works

**Comment by Goldenlion5648** [comment]
sorry, reopened as I see you changed the title to reflect the button being hidden
